.github/workflows/pr-command.yml

```diff
@@ -82,4 +90,5 @@ jobs:
           ```
           $ CRYOSTAT_IMAGE=${{ steps.push-to-ghcr.outputs.registry-path }} sh smoketest.sh
           ```
-      if: github.repository_owner == 'cryostatio'
```

Comment:
Can u do the same thing for other steps in this job? The job's `if` clause already checks this.